core/src/migrations/000086-addDestinationRecordCollection.ts

```diff
@@ -0,0 +1,26 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (
+    queryInterface: Sequelize.QueryInterface,
+    DataTypes: typeof Sequelize
+  ) => {
+    await queryInterface.addColumn("destinations", "collection", {
+      type: DataTypes.STRING(191),
+      allowNull: true,
+    });
+
+    await queryInterface.sequelize.query(
+      `UPDATE "destinations" SET "collection"='group' WHERE "groupId" IS NOT NULL AND state='ready'`
+    );
+
+    await queryInterface.addColumn("runs", "collection", {
+      type: DataTypes.STRING(191),
+      allowNull: true,
+    });
```

Comment:
Is this column being used? It looks like `creatorId` would be the equivalent on the `runs` table